packages/feedback-service/src/types.d.ts

```diff
@@ -1,37 +1,26 @@
-declare module '*.graphql';declare module '*.json';
+declare module '*.graphql';
+declare module '*.json';
 
-export interface IUserTeam {
-    name: string;
-    access: number;
-    primary: Boolean;
-  }
+declare const _;
```

Comment:
No need for this declaration for the `_` type. Install the type definitions for lodash: `@types/lodash`